src/main/resources/kernel_it.properties

```diff
@@ -78,10 +78,10 @@ NLS.seconds = %d Secondi
 NLS.september = Settembre
 NLS.someMinutesAgo = some minutes ago
 NLS.sunday = Domenica
-NLS.thursday = Gioved\u00ec
+NLS.thursday = Giovedì
 NLS.today = Oggi
 NLS.tomorrow = Domani
-NLS.tuesday = arted\u00ec
-NLS.wednesday = Mercoled\u00ec
+NLS.tuesday = artedì
```

Comment:
großes A?

Reply:
not gonna change translations in this commit, only enconding

Reply:
then 11.0-SNAPSHOT^^ and release with next commit / PR